source/cppfront.cpp

```diff
@@ -1664,6 +1664,66 @@ class cppfront
         }
     }
 
+    // Don't work yet, TODO: finalize deducing pointer types from parameter lists
+    auto is_pointer_declaration(parameter_declaration_list_node const* decl_node, int deref_cnt, int addr_cnt) -> bool {
+        return false;
+    }
+
+    auto is_pointer_declaration(declaration_node const* decl_node, int deref_cnt, int addr_cnt) -> bool {
+        if (!decl_node) return false;
```

Comment:
Style: I always enclose my if/else bodies in `{` `}`. So please do the same here and in various other places below. (The reason I do that is mainly to keep myself honest because I require this in Cpp2 (it's the price of not requiring `(` `)` around the conditional-expression) and so I'm forcing myself to use `{` `}` always in Cpp1 too to validate that it's not an onerous requirement, and indeed it is working very well for me.)